src/DependencyInjection/TaskCompilerPass.php

```diff
@@ -1,63 +0,0 @@
-<?php
-
-namespace Task\TaskBundle\DependencyInjection;
-
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-
-/**
- * Compiler pass which collects worker services.
- *
- * @author @wachterjohannes <johannes.wachter@massiveart.com>
- */
-class TaskCompilerPass implements CompilerPassInterface
```

Comment:
@alexander-schranz do you think we should reintroduce that? in my opinion we should pack this functionality into builders (massive/build-bundle) which creates the tasks with an interval on runtime.